pkg/schema/schema.go

```diff
@@ -22,6 +22,9 @@ type DocumentSchema struct {
 	Source     *yamlmeta.Document
 	defaultDVs *yamlmeta.Document
 	Allowed    *DocumentType
+
+	originalLibRef []template.LibRefPiece
+	libRef         []template.LibRefPiece
```

Comment:
is LibRefPiece really related to templating?

Reply:
This also came up in an in-person review we did, yesterday.  👍🏼

Finding/creating the right home for this (and related bits) is on the to-do list, in this issue/PR.

Reply:
We agreed it felt 'off' being in template. We landed on `workspace/ref`.  There was sufficient logic in how libref's are parsed / handled.  We treat this new 'ref' package as a 'go' library used by schemas and datavalues.

Reply:
resolving since this change is pretty outdated. `LibRefPiece` was also renamed to `LibRef`